cpp/src/parquet/types.h

```diff
@@ -668,6 +672,8 @@ PARQUET_EXPORT std::string TypeToString(Type::type t);
 PARQUET_EXPORT std::string FormatStatValue(Type::type parquet_type,
                                            const std::string& val);
 
+PARQUET_EXPORT int32_t GetCompressionCodecDefaultCompressionLevel(Compression::type t);
```

Comment:
Is there a reason to switch to `int32_t`?  It looks like `int` was previously used.

Reply:
I think I used int32_t from the start for everywhere I propagated the compression level. I also used int32_t since it looks like that's the preferred type in Arrow and other programming guidelines.
I know that the compression libraries used in Arrow have the compression level as 'int'. I don't think this is a problem for any system using Arrow but I could replace all int32_t with int if you expect int and int32_t to not have the same size.